docs/guides/build-with-magicavoxel.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Build with MagicaVoxel
+type: guides
+layout: docs
+parent_section: guides
+order: 3
+---
+
+[magicavoxel]: https://ephtracy.github.io/
+
+[MagicaVoxel][magicavoxel] is a free and user-friendly tool that lets you build
```

Comment:
For documentation, teaching is paramount to drawing technical lines. Whatever helps people on their way. If people are constantly running into Slack/SO asking about CORS and web servers, it would serve us well to have guides. I wouldn't think of this as part of the core A-Frame repo, they're resources that happen to live in the repo.

The 3rd party components featured are curated and thoroughly QAed. If A-Frame is going to lean on third-party components, we should promote using them. Especially as we are becoming stricter on what is considered standard. 

We haven't had to do retrospective updates to the docs so far. But if needed, we can promote the `master` version of the workflow such that we can update freely. If there is a critical change that needs to be done on an old version, I am able to point the docs to different commits and tags.